libc/config/linux/x86_64/entrypoints.txt

```diff
@@ -383,6 +383,10 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.cospif
     libc.src.math.dmull
     libc.src.math.dsqrtl
+    libc.src.math.daddl
+    libc.src.math.daddf128
```

Comment:
`daddf128` and `ddivf128` should be under the `LIBC_TYPES_HAS_FLOAT128` guard below.